channeldb/channel_test.go

```diff
@@ -852,6 +852,79 @@ func TestFetchClosedChannels(t *testing.T) {
 	}
 }
 
+// TestFetchWaitingCloseChannels ensures that the correct channels that are
+// waiting to be closed are returned.
+func TestFetchWaitingCloseChannels(t *testing.T) {
```

Comment:
Generally, the fixing commit should be added before the test case, to ensure the commit  history is always building/passing. Not a blocker for this PR tho.